sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/tests/EventHubsTestEnvironment.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Azure.Core.TestFramework;
+
+namespace Microsoft.Azure.WebJobs.Host.EndToEndTests
+{
+    public class EventHubsTestEnvironment: TestEnvironment
```

Comment:
Why not pull in the `Testing` projitems from the Event Hubs shared project?  That'd give you the full test environment used in the SDK.

Reply:
It uses some internal types that are not visible to the WebJobs projects

Reply:
I don't think we want an IVT from the EH to WebJobs extensions.